docs/installation.md

```diff
@@ -318,9 +318,10 @@ reference documentation](https://github.com/trinodb/charts/blob/main/charts/gate
 
 ### Health Checks
 
-Trino Gateway checks the health of each backend and **deactivates it if 
-unhealthy**. A backend that fails a health check must be manually reset to 
-active. Automatic recovery is not supported.
+Trino Gateway checks the health of each backend and maintains an in-memory state for each backend.
+A backend that fails the health check will be set to UNHEALTHY and trino gateway will not route to
+such backends. This is independent of the active/inactive status. 
```

Comment:
do we have docs on what active/inactive status mean? if we do, how about adding a hyperlink to this.

Reply:
no.. i don't think so. We do have other references on `active`, but we never really have a proper definition for it